jetson-nano.coffee

```diff
@@ -38,7 +38,7 @@ module.exports =
 	configuration:
 		config:
 			partition:
-				primary: 1
+				primary: 12
```

Comment:
That should be `partition: 12`, there's no primary partitions on gpt partitioned disks.
And even for MBR disks, we can just use the partition index instead of { primary, logical }.

Reply:
@zvin interesting, so the TX2 is different partiton-wise? There it is set up the same:

https://github.com/balena-os/balena-jetson-tx2/blob/db302da56d368f5c9c91d3de496144484f37b01e/jetson-tx2.coffee#L49-L53

Reply:
Yes, tx2 image partitions:
```
Model: Loopback device (loopback)
Disk /dev/loop0: 3259MB
Sector size (logical/physical): 512B/512B
Partition Table: gpt
Disk Flags: 

Number  Start   End     Size    File system  Name         Flags
 1      4194kB  88.1MB  83.9MB  fat16        resin-boot   boot, esp
 2      88.1MB  3234MB  3146MB  ext4         resin-rootA
 3      3234MB  3238MB  4194kB  ext4         resin-rootB
 4      3238MB  3242MB  4194kB               resin-state
 5      3242MB  3258MB  15.7MB               resin-data
```

Reply:
Ack, and makes sense as discussed in [Flowdock](https://www.flowdock.com/app/rulemotion/resin-devices/threads/lLhZMo4MnETGR7OTPfzBeC5Zsd3). Updated PR.